lib/stencil-bundle.js

```diff
@@ -152,26 +154,35 @@ class Bundle {
         };
     }
 
-    async assembleTemplatesTask(callback) {
-        console.log('Template Parsing Started...');
-        const internalTemplatesList = await recursiveReadDir(this.templatesPath, ['!*.html']);
-        const searchExternalLib = (content) => {
-            const externalPathRegex = /{{2}>\s*(external)[^{]*?}{2}/g;
-            const externalTemplatesImports = content.match(externalPathRegex);
-            if (!externalTemplatesImports) return [];
+    /**
+     * helps to find any node modules dependencies with
+     * ui templates and returns its list
+     *
+     * @private
+     * @param {string} content
+     * @returns {Array}
+     */
+    async _getExternalLibs(templatePath) {
+        const content = await fs.promises.readFile(templatePath, { encoding: 'utf-8' });
+        const externalPathRegex = /{{2}>\s*(external)[^{]*?}{2}/g;
+        const externalTemplatesImports = content.match(externalPathRegex);
 
-            return externalTemplatesImports.map((_import) => _import.split('/')[1]);
-        };
-        const checkTemplate = async (templatePath) => {
-            const content = await fs.promises.readFile(templatePath, { encoding: 'utf-8' });
-            const res = searchExternalLib(content);
+        if (!externalTemplatesImports) return [];
 
-            return res;
-        };
+        return externalTemplatesImports.map((_import) => _import.split('/')[1]);
```

Comment:
Is there a reason using underscore in `_import`?

Reply:
do you have any good name suggestions?

Reply:
np, updated